app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  has_many :reservations, dependent: :destroy
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   enum role: %i[user manager admin]
```

Comment:
consider an explicit enum
enum role: { user: 0, manager: 1, admin: 2 }
we then know what the database fields are more explicitly